openedx/core/djangoapps/user_authn/views/tests/test_logout.py

```diff
@@ -0,0 +1,115 @@
+"""
+Tests for logout
+"""
+import unittest
+
+import ddt
+from django.conf import settings
+from django.test import TestCase
+from django.urls import reverse
+from edx_oauth2_provider.constants import AUTHORIZED_CLIENTS_SESSION_KEY
+from edx_oauth2_provider.tests.factories import (
+    ClientFactory,
+    TrustedClientFactory
+)
+from student.tests.factories import UserFactory
```

Comment:
@nasthagiri: I think we are ok with importing from student for now for things like this, but just checking.  Maybe we should add a clean-up comment about moving the UserFactory?

Reply:
How do you suppose a comment here will help?

Reply:
I meant in JIRA.  I added a bullet to ARCH-204.